README.md

```diff
@@ -1,14 +1,14 @@
 # prefactor-cwl
 CWL version of prefactor
 
-https://github.com/lofar-astron/prefactor/
+https://github.com/EOSC-LOFAR/prefactor-cwl
```

Comment:
this is wrong, it was a link to the original prefix repo.

Reply:
D'oh, mea culpa